webrender/src/box_shadow.rs

```diff
@@ -207,6 +208,15 @@ impl FrameBuilder {
                         width = 2.0 * max_width + BLUR_SAMPLE_SCALE * blur_radius;
                         height = 2.0 * max_height + BLUR_SAMPLE_SCALE * blur_radius;
 
+                        // If the width or height ends up being bigger than the original
+                        // primitive shadow rect, just blur the entire rect and draw that
+                        // as a simple blit.
+                        if width > prim_info.rect.size.width || height > prim_info.rect.size.height {
```

Comment:
is it excluding `corner_size.is_some()` case by intention?

Reply:
Yep, that's handled in the branch above this one.